jenkins_sandbox/plugins.txt

```diff
@@ -0,0 +1,177 @@
+jira-steps
```

Comment:
These are all going to need pinned versions. We can generate that from the current master with some groovy.